joblib/func_inspect.py

```diff
@@ -80,6 +88,7 @@ def _clean_win_chars(string):
         quote = urllib.quote
     else:
         # In Python 3, quote is elsewhere
+        import urllib.parse
```

Comment:
Just to share the WTF moment I just had, the same two lines of code work fine inside IPython:

```
Python 3.4.3 |Continuum Analytics, Inc.| (default, Mar  6 2015, 12:03:53) 
Type "copyright", "credits" or "license" for more information.

```
IPython 3.1.0 -- An enhanced Interactive Python.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://binstar.org
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import urllib

In [2]: urllib.parse
Out[2]: <module 'urllib.parse' from '/volatile/le243287/miniconda3/lib/python3.4/urllib/parse.py'>
```

I am guessing somewhere in the IPython code, `urllib.parse` is imported ...